diff --git a/lib/difflib.js b/lib/difflib.js
index 80d250e7e18bdc972df3621ee5c05ffff0e3659f..c51e0282cc6b535fb288bd78ff28740b08c2d9c3 100644
--- a/lib/difflib.js
+++ b/lib/difflib.js
@@ -75,8 +75,6 @@ Class Differ:
 
   SequenceMatcher = (function() {
 
-    SequenceMatcher.name = 'SequenceMatcher';
-
     /*
       SequenceMatcher is a flexible class for comparing pairs of sequences of
       any type, so long as the sequence elements are hashable.  The basic
@@ -737,8 +735,6 @@ Class Differ:
 
   Differ = (function() {
 
-    Differ.name = 'Differ';
-
     /*
       Differ is a class for comparing sequences of lines of text, and
       producing human-readable differences or deltas.  Differ uses
